### PR TITLE
Update issue templates

### PR DESCRIPTION
**Pull request by tim-pugh**

### CODE REVIEW

**Comment by tim-pugh** [comment]
This is simply a issue template provided by github that may be useful for us to craft up writing them up.

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
+
+**Desktop (please complete the following information):**
+ - OS: [e.g. iOS]
+ - Browser [e.g. chrome, safari]
```
Comment (connorkuehl) [inline .github/ISSUE_TEMPLATE/bug_report.md]: We could safely remove the browser field here.

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+ - Version [e.g. 22]
+
+**Smartphone (please complete the following information):**
```
Comment (connorkuehl) [inline .github/ISSUE_TEMPLATE/bug_report.md]: I think we can remove the Smartphone section as well.

**Comment by tim-pugh** [comment]
Ready to be merged.

**Comment by nforbus** [comment]
Can't tell what it does until its merged, but it doesn't affect other files so its harmless for merge.

**Pull request merged**